mathlibtools/fetching.py

```diff
@@ -8,7 +8,12 @@
 from mathlibtools.auth_github import auth_github
 from mathlibtools.delayed_interrupt import DelayedInterrupt
 
-def mathlib_asset(rev):
+
+# this should check if a file exists at url without actually downloading it
+def archive_exists_at_url(url):
+    return False
```

Comment:
I don't know the Python3 way to do this and don't have time right now to search -- wanted to make this PR before leaving the office.